package-lock.json

```diff
@@ -5,7 +5,7 @@
   "requires": true,
   "packages": {
     "": {
-      "version": "2.1.0",
```

Comment:
Let's not commit changes into this file

Reply:
Sorry, I didn't notice that the file had a lot of changes